crates/xtask/src/codegen.rs

```diff
@@ -99,7 +99,9 @@ impl CodegenCommand {
     fn generate_type_bindings<P: AsRef<Path>>(header_file: P) -> Result<bindgen::Bindings> {
         bindgen::Builder::default()
             .header(header_file.as_ref().to_string_lossy())
-            .allowlist_type("ie_.*")
+            .clang_arg("-I/home/rahul/repos/openvino-binding/openvino-rs/crates/openvino-sys/upstream/src/bindings/c/include")
```

Comment:
This needs to be a relative path, maybe `-Iopenvino-sys/upstream/src/bindings/c/include`

Reply:
done, changed to `.clang_arg("-I./crates/openvino-sys/upstream/src/bindings/c/include")`